devise-a-value-proposition/readme.md

```diff
@@ -0,0 +1,31 @@
+> Goal of this guide: understanding a value proposition.
+
+# How to Devise a Value Proposition
+
+***A value proposition is a promise of value to be delivered. It’s the primary reason a prospect should buy from you/use your product.***
+
+## What makes a good value proposition:
+
+* Clarity! It’s easy to understand.
+* It communicates the concrete results a customer will get from purchasing and using your products and/or services.
+* It says how it’s different or better than the competitor’s offer.
+* It avoids hype (like ‘never seen before amazing miracle product’), superlatives (‘best’) and business jargon (‘value-added interactions’).
+* It can be read and understood in about 5 seconds.
+
+### A value proposition usually takes the form of a tagline
+
+A good tagline:
+
+* Shows brand awareness
+* Creates an emotional response that sticks in people’s minds long after it’s been seen or heard
+* Requires a lot of insight into the company and its people
+* Is original
+* Is concise
+* Is relevant
+* Has staying power (is not trendy)
+* Is in the language of the end-user
+
```

Comment:
Perhaps this should be in the guide how-to-write-a-good-tagline? And than link to that?